packages/ember-htmlbars/tests/helpers/with_test.js

```diff
@@ -281,122 +32,3 @@ QUnit.module('{{#with}} helper binding to view keyword', {
 QUnit.test('{{with}} helper can bind to keywords with \'as\'', function() {
```

Comment:
In old views, the template's context was the value of its `context` property. Components hardcode that to the component itself. In fact, the initial implementation of components was very simple: just set the `controller` and `context` properties to `this` (see [my initial implementation](https://github.com/emberjs/ember.js/blob/c52e4200b57c399192582598e991accd3332b44a/packages/ember-views/lib/views/component.js))

In Ember 2.0, this semantic is now an intrinsic part of components; changing `component.context` has no effect. These tests are testing views, which are still supported through the legacy view helper, but our test harness is limited to components (which will be supported first-class in the new rendering engine).

As a result, you do not need to port any tests that rely on view-specific behavior to the new harness (but you should not delete them, since we still support the behavior through the addon). In general, if you find tests like these, you might also want to investigate whether the test is only using the legacy behavior accidentally, but really means to test something else.